contrib/verify-commits/trusted-keys

```diff
@@ -2,3 +2,4 @@
 3F1888C6DCA92A6499C4911FDBA1A67379A1A931
 32EE5C4C3FA15CCADB46ABE529D4BCB6416F53EC
 FE09B823E6D83A3BC7983EAA2D7F2372E50FE137
+FE09B823E6D83A3BC7983EAA2D7F2372E50FE137
```

Comment:
I know I can be number-blind sometimes, but isn't this the same as the entry above it?